<?php

namespace ContainerENcUe0a;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class getGenerateSuspiciousTableService extends App_KernelDevDebugContainer
{
    /**
     * Gets the private 'App\HL\Command\GenerateSuspiciousTable' shared autowired service.
     *
     * @return \App\HL\Command\GenerateSuspiciousTable
     */
    public static function do($container, $lazyLoad = true)
    {
        include_once \dirname(__DIR__, 4).'/vendor/symfony/console/Command/Command.php';
        include_once \dirname(__DIR__, 4).'/src/HL/Command/GenerateSuspiciousTable.php';
        include_once \dirname(__DIR__, 4).'/src/HL/Application/GetClientSuspicious.php';
        include_once \dirname(__DIR__, 4).'/src/HL/Domain/ClientSuspiciousRepository.php';
        include_once \dirname(__DIR__, 4).'/src/HL/Infrastructure/CsvClientSuspiciousRepository.php';

        $container->privates['App\\HL\\Command\\GenerateSuspiciousTable'] = $instance = new \App\HL\Command\GenerateSuspiciousTable(new \App\HL\Application\GetClientSuspicious(new \App\HL\Infrastructure\CsvClientSuspiciousRepository()));

        $instance->setName('app:generate-suspicious-table');

        return $instance;
    }
}
